layers/+lang/clojure/README.org

```diff
@@ -76,50 +77,101 @@ Or set this variable when loading the configuration layer:
   '((clojure :variables clojure-enable-fancify-symbols t)))
 #+END_SRC
 
-** CIDER and clj-refactor
-Note that recent versions of CIDER automatically inject the required
-dependencies into your boot or leiningen configuration when you connect to the
-REPL. Thus, the configuration instructions below only apply to CIDER 0.10 and
-older. Most users should be able to just run ~SPC m s i~ to connect to the CIDER
-REPL and skip the rest of this section.
+** Enabling sayid or clj-refactor
+The packages sayid (Clojure debugger) and clj-refactor (automatic refactorings)
+are disabled by default. To enable them, add a `:variables` option when enabling
+the `clojure` layer, specifying `clojure-enable-sayid`, or
+`clojure-enable-clj-refactor`, or both.
+
+In your Spacemacs configuration:
+
+#+BEGIN_SRC emacs-lisp
+  ;; before
+  dotspacemacs-configuration-layers
+  '(...
+    clojure
+   )
+
+  ;; after
+  dotspacemacs-configuration-layers
+  '(...
+    (clojure :variables clojure-enable-sayid t
+                        clojure-enable-clj-refactor t)
+   )
```

Comment:
It seems to be a general convention to list the names of layer variables under :variables 

    (clojure :variables 
                 clojure-enable-sayid t
                 clojure-enable-clj-refactor t)

Although this could be just my convention - https://gist.github.com/jr0cket/90c415dbc268be32b3ffe16bf40012fc#file-spacemacs-el-L42-L86